h2integrate/resource/wind/test/test_openmeteo_wind_api.py

```diff
@@ -0,0 +1,143 @@
+    plant = {
+        "plant_life": 30,
+        "simulation": {
+            "dt": 3600,
+            "n_timesteps": 8760,
+            "start_time": "01/01/1900 00:30:00",
+            "timezone": 0,
+        },
+    }
+    return plant
+
+
+@fixture
+def plant_simulation_nonutc_start():
+    plant = {
+        "plant_life": 30,
+        "simulation": {
+            "dt": 3600,
+            "n_timesteps": 8760,
+            "start_time": "01/01/1900 00:30:00",
+            "timezone": -6,
+        },
+    }
+    return plant
+
+
+@fixture
+def site_config_download_from_web():
+    site = {
+        "latitude": 44.04218,
+        "longitude": -95.19757,
+        "resources": {
+            "wind_resource": {
+                "resource_model": "openmeteo_wind_api",
+                "resource_parameters": {
+                    "resource_year": 2023,
+                    "resource_filename": "open-meteo-44.04N95.20W438m.csv",
+                },
+            }
+        },
+    }
+    return site
+
+
+@fixture
+def site_config_download_from_h2i():
+    site = {
+        "latitude": 44.04218,
+        "longitude": -95.19757,
+        "resources": {
+            "wind_resource": {
+                "resource_model": "openmeteo_wind_api",
+                "resource_parameters": {
+                    "resource_year": 2023,
+                    # "resource_filename":
+                    # "44.04218_-95.19757_2023_openmeteo_archive_60min_local_tz.csv",
+                },
+            }
+        },
+    }
+    return site
+
+
+def test_wind_resource_web_download(
+    plant_simulation_utc_start, site_config_download_from_web, subtests
+):
+    plant_config = {
+        "site": site_config_download_from_web,
+        "plant": plant_simulation_utc_start,
+    }
+
+    prob = om.Problem()
+    comp = OpenMeteoHistoricalWindResource(
+        plant_config=plant_config,
+        resource_config=plant_config["site"]["resources"]["wind_resource"]["resource_parameters"],
+        driver_config={},
+    )
+    prob.model.add_subsystem("resource", comp)
+    prob.setup()
+    prob.run_model()
+    wind_data = prob.get_val("resource.wind_resource_data")
+
+    with subtests.test("filepath for data was found where expected"):
+        assert Path(wind_data["filepath"]).exists()
+        assert Path(wind_data["filepath"]).name == "open-meteo-44.04N95.20W438m.csv"
+
+    data_keys = [k for k, v in wind_data.items() if not isinstance(v, (float, int, str))]
+    with subtests.test("Data timezone"):
+        assert pytest.approx(wind_data["data_tz"], rel=1e-6) == 0
+    with subtests.test("Site Elevation"):
+        assert pytest.approx(wind_data["elevation"], rel=1e-6) == 438
+    with subtests.test("resource data is 8760 in length"):
+        assert all(len(wind_data[k]) == 8760 for k in data_keys)
+    with subtests.test("theres 12 timeseries data keys"):
+        assert len(data_keys) == 12
+
+
+def test_wind_resource_h2i_download(
```

Comment:
how is this different than the previous test? Do we need both? Could they be lumped into a single test with both downloads checked?